libbeat/common/uuid.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+// GenerateUUID is used to generate a random UUID
+// This was copied from https://github.com/hashicorp/uuid/blob/master/uuid.go
```

Comment:
copied code license is Mozilla License.

Please let's not copy others code and either.

We want RFC compatible UUIDs? If so, which uuid version is this? Looks like 4 (all random), but missing version number.